components/list/list-item-generic-layout.js

```diff
@@ -65,6 +65,8 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			::slotted([slot="content"]),
 			::slotted([slot="actions"]) {
```

Comment:
I think if you pull the actions slot out on its own without the overflow and position styling it prevents the tooltips and expanded menu from getting covered up.

Reply:
Yeah we definitely need to be careful of overflow with the dropdown components.  The dropdown has some magic positioning, and if you place it in a container that does not allow overflow, there may be some issues with it being cut off or not displaying correctly.

Reply:
Yup! I totally forgot about this PR :( I'll fix the drop down issue and only hide on the overflow-x